dev/misc_utils/git_jira_fix_version_check.py

```diff
@@ -91,11 +91,15 @@
 
 print('\nAny resolved Jira with fixVersion ' + fix_version
       + ' but corresponding commit not present')
-print('Starting diff: ##############################################')
+
+# maxResults=None results in internal batching within the API 
+# and it returns all the results back
 all_issues_with_fix_version = jira.search_issues(
     'project=' + jira_project_name + ' and status in (Resolved,Closed) and fixVersion='
-    + fix_version)
+    + fix_version, maxResults=None)
+print(f'Found {all_issues_with_fix_version.total} issues fixed in {fix_version}')
```

Comment:
minor comment : Looks like the formatted string literal is supported from python3.6 onwards - https://docs.python.org/3/whatsnew/3.6.html#whatsnew36-pep498. 

Is there any minimum python version support wrt the phoenix repo where the current script should run ? As this might not be backward compatible with lower versions.

Reply:
@gourabtaparia This script only works with python3. Python 3.6 is EOL so we should be fine.